src/01_clock_resolution.jl

```diff
@@ -21,8 +21,11 @@ function estimate_clock_resolution(n_samples::Integer = 10_000)
     for _ in 1:n_samples
         t1 = Base.time_ns()
         t2 = Base.time_ns()
-        Δt = t2 - t1
-        min_Δt = min(min_Δt, Δt)
+        # On Linux AArch64 it seems that t1 and t2 could be the same sometimes
+        if t2 > t1
+            Δt = t2 - t1
+            min_Δt = min(min_Δt, Δt)
+        end
     end
 
     min_Δt
```

Comment:
Yeah, I relized that, the code is longer which is why I didn't do that. I'll update it ...